docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: '2'
 
 services:
   goaccess:
+    build: .
```

Comment:
Docker Hub has turned off automated builds more than year ago, so building in local context should be fine.